react/LocationSearch.tsx

```diff
@@ -1,50 +1,92 @@
 import React, { useState } from 'react'
-import { Input, IconSearch, IconClear } from 'vtex.styleguide'
 import { FormattedMessage } from 'react-intl'
+import { Input, IconSearch, IconClear } from 'vtex.styleguide'
+import {
+  Combobox,
+  ComboboxInput,
+  ComboboxOption,
+  ComboboxPopover,
+  ComboboxList,
+} from '@reach/combobox'
+import '@reach/combobox/styles.css'
 
+import { addresses as mockedAddresses } from './addresses'
 import styles from './LocationSearch.css'
 
-const LocationSearch: React.FC = () => {
-  const [inputValue, setInputValue] = useState<string>('')
+const useAddressMatch = (term: string): string[] => {
+  return mockedAddresses
+    .filter((address: string) =>
+      address.toLocaleLowerCase().includes(term.toLocaleLowerCase())
+    )
+    .slice(0, 6)
```

Comment:
why these arguments? can you extract them to a constant or add a comment?

Reply:
didn't put too much thought into it, as I supposed that this should be handled on the GraphQL query. Will add a comment

Reply:
I got confused with this (read too fast haha), but this is to limit the number of addresses shown in the dropdown right? we could convert the 6 to a constant like `MAX_DROPDOWN_ADDRESSES` or something

Reply:
Yes. And sure, the constant will definitely make it easier to read, thks.